src/components/interfaces/HMI_API.xml

```diff
@@ -1917,11 +1917,11 @@
        True: Available, False: Not Available, Not present: Not Available.
      </description>
    </param>
-   <param name="availableHDsAvailable" type="Boolean" mandatory="false" >
+   <param name="availableHdChannelsAvailable" type="Boolean" mandatory="false">
      <description>
-       Availability of the getting the number of available HD channels.
+       Availability of the list of available HD sub-channel indexes.
        True: Available, False: Not Available, Not present: Not Available.
-     </description>
+    </description>
```

Comment:
@IGapchuk please keep xml formatting

Reply:
@LuxoftAKutsan Unit test will added by separate commit.
XML formatting fixed in 39d969d46ff3099f811e17443222a2af87425a20.